modules/copy-template/build/copy-template/Dockerfile

```diff
@@ -1,14 +1,9 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal AS builder
-RUN microdnf install -y tar gzip && microdnf clean all
+FROM registry.access.redhat.com/ubi9/go-toolset:1.18.9 AS builder
```

Comment:
The `go.mod` file of `copy-template` task has already moved to `go 1.19`, so using this older version as the builder can fail tthe build when using 1.19 features. Can't the builder use 1.19 too?